src/commands/Minion/aerialfish.ts

```diff
@@ -0,0 +1,203 @@
+	}
+];
+
+const sellables = [
+	{
+		item: getOSItem('Golden tench'),
+		cost: 100,
+		aliases: []
+	}
+];
+
+export async function alryChat(str: string) {
+	const image = await chatHeadImage({
+		content: str,
+		name: 'Alry the Angler',
+		head: 'alryTheAngler'
+	});
+	return new MessageAttachment(image);
+}
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			altProtection: true,
+			oneAtTime: true,
+			usage: '[buy|sell|run] [name:...string|tripTime:int{1}]',
+			subcommands: true,
+			usageDelim: ' ',
+			cooldown: 1,
+			aliases: ['aerial', 'af'],
+			description: 'Sends your minion to aerial fish, allowing you to get the angler outfit.',
+			examples: ['+aerialfishing 30', '+aerialfishing buy fish sack'],
+			categoryFlags: ['minion', 'skilling']
+		});
+	}
+
+	@minionNotBusy
+	@requiresMinion
+	async run(
+		msg: KlasaMessage,
+		[tripTime = Math.floor(msg.author.maxTripLength / Time.Minute)]: [number]
+	) {
+		await msg.author.settings.sync(true);
+
+		if (
+			msg.author.skillLevel(SkillsEnum.Fishing) < 43 ||
+			msg.author.skillLevel(SkillsEnum.Hunter) < 35
+		) {
+			return msg.send(
+				`You need atleast level 35 Hunter and 43 Fishing to do Aerial fishing.`
+			);
+		}
+
+		let tripLength = Time.Minute * tripTime;
+
+		if (tripLength > msg.author.maxTripLength) {
+			return msg.send(
+				`${msg.author.minionName} can't go on trips longer than ${formatDuration(
+					msg.author.maxTripLength
+				)}, try a lower trip length. The highest amount of minutes you can send out is ${Math.floor(
+					msg.author.maxTripLength / Time.Minute
+				)}.`
+			);
+		}
+
+		const randValue = randFloat(1.85, 2.15);
+		const quantity = tripLength / (randValue * Time.Second);
+		const duration = quantity * (randValue * Time.Second);
+
+		await addSubTaskToActivityTask<AerialFishingActivityTaskOptions>(this.client, {
+			userID: msg.author.id,
+			channelID: msg.channel.id,
+			quantity,
+			duration,
+			type: Activity.AerialFishing
+		});
+
+		return msg.send(
+			`${
+				msg.author.minionName
+			} is now doing Aerial fishing, it will take around ${formatDuration(
+				duration
+			)} to finish.`
+		);
+	}
+
+	@requiresMinion
+	async buy(msg: KlasaMessage, [itemName = '']: [string]) {
+		const buyable = buyables.find(
+			i =>
+				stringMatches(itemName, i.item.name) ||
+				i.aliases.some(alias => stringMatches(alias, itemName))
+		);
+
+		if (!buyable) {
+			return msg.send(
+				`Here are the items you can buy: \n\n${buyables
+					.map(i => `**${i.item.name}:** ${i.cost} Molch pearls`)
+					.join('\n')}.`
+			);
+		}
+		await msg.author.settings.sync(true);
+		const bank = msg.author.bank();
+		const amountPearlsHas = bank.amount('Molch pearl');
+		if (amountPearlsHas === 0) {
+			return msg.send(
+				await alryChat(
+					`You have no Molch pearls, but here is joke, Where do fish keep their money? \nIn a riverbank. Hehe!`
```

Comment:
``You have no Molch pearls, but here is a joke, where do fish keep their money? \nIn a riverbank. Hehe!``